File: v2/android/src/main/java/io/swagger/client/model/Advisorresponse.java
package io.swagger.client.model;

import io.swagger.client.model.Advisor;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Advisorresponse  {
  
  @SerializedName("advisor")
  private Advisor advisor = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Advisor getAdvisor() {
    return advisor;
  }
  public void setAdvisor(Advisor advisor) {
    this.advisor = advisor;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Advisorresponse {\n");
    
    sb.append("  advisor: ").append(advisor).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Advisorresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Advisorresponse {
    

    
    public Advisor Advisor { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Advisorresponse {\n");
      
      sb.Append("  Advisor: ").Append(Advisor).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}